### ISSUE TITLE
[Suggestion] Changing the topic "being a vegetarian"

### ISSUE THREAD

**Issue by eltulax** [Monika-After-Story/MonikaModDev#7227]
I think that it would be nice to change this topic, because I see some problems with it.

1. It has some points which do not make sense or are excessive.
2. Other points let Monika seem hypocritical if you compare them to other topics.
3. Not really important, but this topic hurt my feelings and I think she is not really respectful especially because of the first problem.

1
The statement that it is murder when an animal is "just" a little bit bigger than an insect ignores some facts. Firstly there is a big difference in intelligence, self-awareness and emotional capability between insects and mammals, so it is much easier and more logical to have empathy towards mammals than towards some insects.
Secondly it does not only matter the act of killing. I personally understand hunting animals to protect forests etc., but the planned animal breeding in factory farming, where the animals live under really bad conditions until the finally get killed is just cruel for me and not even close to killing an insect. And the point with killing microorganisms does not make sense either. Nobody kills them intentionally or only because it is necessary to protect humans against diseases etc.
The statement that plants could also have feelings is weird. Even if it were true, you would generate much more pain in the world when you feed the animals with plants and afterwards kill the animal to get the meat, instead of eating the plants yourself in the first place.
Also it is weird that Monika is a vegetarian at all, because her only reason is the environment, so it would not be a problem to eat meat when it would become waste otherwise.

2
In the topic "Owning pets" Monika mentions that she would feel really bad if she kept a Resplendent Quetzal to be a pet. So it is ethically wrong to keep a bird in captivity, but factory farming is alright, where the same thing happens with the intention to kill the animals? This seems hypocritical for me.

3
I am probably biased, but I think the way she speaks in this topic is not respectful, because she just bashes on people that are vegetarians because of ethical reasons and thinks that this reason is really stupid and irrational.

I think the topic would be better if it were more general and less negative. I think it should not matter why a person is vegetarian. Also asking the player whether he is vegan, vegetarian, trying to be one or just eats meat would be a nice addition.

To sum up, I think the topic does not really match with Monika's personality, because she seems very cold, with many bad arguments that are not thought through and she seems really biased herself.
I do not want to offend anyone with this suggestion or be disrespectful myself, so I would of course discuss about it or change sentences, if it offended someone. It would be nice, if you thought about this suggestion.

**Comment by MrVavlo** [comment]
Not going to agree or disagree with you there.
I just want to point out that quetzals are near-threatened species, and _this_ is the reason why she would feel bad to have them as a pet.

And also, isn't the topic in subject is from DDLC itself? _And it doesn't match her personality, huh..._

**Comment by multimokia** [comment]
Monika too, is entitled to her own opinions and thoughts on these subjects. Whether or not you agree with them shouldn't change how she feels about it.

In other words, her points *are* valid whether you agree or not.

Though I can't help but feel you might have misread the topics too, I don't believe Monika ever explicitly stated she agreed with factory farming.

Also as @MrVavlo pointed out, this is an original act 3 topic, in other words, this was part of her from the very beginning. Though even if it weren't, the above points still apply.

I don't think she should just change her opinions on things just because her player disagrees with them, that'd just make her more of a flat and 2-dimensional character.

**Comment by eltulax** [comment]
> Monika too, is entitled to her own opinions and thoughts on these subjects. Whether or not you agree with them shouldn't change how she feels about it.

Of course she does, but I think that she is disrespectful about my position and that she makes "fun" about it.

> Though I can't help but feel you might have misread the topics too, I don't believe Monika ever explicitly stated she agreed with factory farming.

She says that she does not care about being inhumane and all that, so I interpreted it that she does not really care how the animals are treated.

> Also as @MrVavlo pointed out, this is an original act 3 topic, in other words, this was part of her from the very beginning. Though even if it weren't, the above points still apply.

Yes I know it is an original topic, but I think that she acts very differently than in other topics. I always thought that she tries to be as rational and logical as possible and also she tries to be not offensive at all with the exception of the other girls. But maybe I am the only one who thinks so and therefore must accept it.

**Comment by Freshairkaboom** [comment]
Though I agree that the topic is a bit cold and doesn't take into account the feelings of those who feel different about the ethics of killing animals for food, that doesn't mean it's not part of Monika's personality. Dan Salvato saw fit to write that bit of Monika in there, so that's what we're dealing with when it comes to canon material. 

One does not have to agree with Monika on everything, I certainly don't. But I love her nonetheless. And at least while she's in there, she won't be buying any animal products regardless, so that's a piece of solace for me.

While she's well read on a range of topics, we must also remember that her world is not our world. It doesn't actually contain any animals in it, so whatever position one would have on the eating of animal products would be entirely inconsequential anyway, it's simply a part of her character backstory. Her actual interaction with animals is zero, and who knows how she would feel seeing a real one after "crossing over"?

But a lot of hurdles will have to be leapt through for such an event to take place anyway. By that time the world and how we treat animals may be very different to how it is now.

**Comment by Booplicate** [comment]
> there is a big difference in intelligence, self-awareness and emotional capability between insects and mammals, so it is much easier and more logical to have empathy towards mammals than towards some insects

That's an opinion, but it doesn't mean that everyone should care about killing an animal for food. Her position is that either way it's murder, and she's (probably) ok with that if it's done for the said reasons.

> you would generate much more pain

You can't "generate" pain. Her point is about why having empathy towards animals, but not towards plants.

> she would feel really bad if she kept a Resplendent Quetzal

Keeping nearly-extinct species for yourself only for its aesthetics is one thing, breeding animals for food is another. For example, I think she would be against hunting "for fun" or just for "trophies".

> she just bashes on people

I can see why you'd think so. But she isn't saying she's the only one right and you're wrong, she just "finds it weird".

And from the last message
> she tries to be as rational and logical as possible and also she tries to be not offensive

She's still rational, just having different view on the problem. She may come off as rude in some of her conversations, and she even may make mistakes. But don't all people do? And then, she does respect your position anyway.

I hope that helps enlighten her point a little. I think it's actually good that she has her points and doesn't just agree on whatever the player says. Different people have different positions. And you don't have to come to an agreement on everything to be together. Plus, in the end you both are doing the same thing, having one goal, just by different reasons.

That all, and we don't change the original topics. She is what she is, MAS Monika is the same Monika you saw in DDLC.

**Comment by CloakedGhost** [comment]
She said she "doesn't care as much", meaning she does care but only to a certain degree. The difference with the Quetzal is that the bird would be suffering in her presence, meaning it's much worse when it happens around you. Humans are suffering all over the world too, but we can only care to a certain degree. If we cared so much for everyone in the world, we would be crying and suffering non stop. It makes a huge difference if something bad happens to another human/an animal around you and not somewhere in the world where you don't see it, that's just human nature. 

And she is not trying to offend you or make fun of you, she is just questioning things. This is one of the cases where I wish full on discussions with her were possible, because she might actually learn something from you (it's what she said, stay willing to learn at all times). But since we can't do that, I think it's alright to keep the topic this way and let her have her opinion.

**Comment by Wingdinggaster656** [comment]
Personally, I don't agree.

There's never anything wrong with being a vegetarian, and there's never anything wrong with not wanting to change for player.

Not to mention that Monika doesn't actually "hate" meat that much, a question the player can ask is "Would you ever eat meat?"

Monica's response to this question was: "I would, but I don't want meat to be my main diet." I don't think it's bad to simply not want meat to be the main food. In fact, there are many people who do not eat meat as their main food.

**Comment by Cosmic-Incantations** [comment]
I mean, I agree that there should be (some) changes..... I did think it was hypocritical of her for all those points
maybe instead she could skip out on the environment part or something and could talk about the bigger problems

but she might already not be vegetarian cause she said she just dose not _care_ about meat, but she would eat it
if she was, she would say that meat would make her sick, maybe she is more casual about meat but doesn't eat it often

**Comment by kkrosie123** [comment]
This is dialogue from the base game. We do not intend to change it or break away from the core writing we have based this entire mod on.
